x/ccv/consumer/keeper/changeover.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func (k Keeper) ChangeoverToConsumer(ctx sdk.Context) (initialValUpdates []abci.ValidatorUpdate) {
```

Comment:
Add some comments on what this function does.

Reply:
See https://github.com/cosmos/interchain-security/commit/a07e84bac7ac8bcdfec8503911cae8fea0b50285